database/migrations/2018_07_16_173334_CreateTeamMembersTable.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateTeamMembersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('team_members', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('rollNo')->unsigned();
+            $table->string('team_id');
```

Comment:
I think this is a foreign key. Add a constraint 
`$table->foreign('team_id')->references('id')->on('teams');`

You can make `team_id` an `integer`